lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -154,13 +163,17 @@ serveWallet (cfg, tr) databaseDir hostPref listen lj beforeMainLoop = do
                         & setBeforeMainLoop (beforeMainLoop sockAddr nPort bp)
                 let ipcServer = daedalusIPC tracerIPC wPort
                 let apiServer =
-                        Server.start settings tracerApi socket rndWallet seqWallet
+                        Server.start
+                            settings tracerApi socket rndWallet seqWallet spl
                 race_ ipcServer apiServer
                 pure ExitSuccess
             Left e -> handleApiServerStartupError e
 
     toWLBlock = J.convertBlock
 
+    poolDBPath :: Maybe FilePath -> Maybe FilePath
+    poolDBPath = fmap (<> "/stake-pool-metrics.sqlite")
```

Comment:
`stake-pool-metrics.sqlite` --> `stake-pools`

Reply:
Why?

I chose `stake-pool-metrics.sqlite` to highlight that the purpose is not to simply "store stake-pools" (what does that even mean / why would we need to do that anyway?), but to collect metrics about them.

What do you think?

Reply:
Well, I'd rather be a bit more general with the file name here. I don't like to "code for the future", but for things that live at the boundaries of the system, we have to be a bit careful. "stake-pools" is a bit more general, and opens up for extension. In the end, there's no need to be that precise at this stage. 

As for the `.sqlite` extension, my bad, I thought this was added automatically by persistent. Still, I'd prefer to have the name for the database part of the `Pool.DB.Sqlite` module, very much like we do for the wallets databases.

Reply:
Something factory-like felt overkill. But I moved this function to `Cardano.Pool.DB.Sqlite` (but without Maybe).